src/metabase/query_processor.clj

```diff
@@ -74,13 +74,13 @@
   ([query]
    (process-query query nil))
 
-  ([query :- ::qp.schema/query
+  ([query :- ::qp.schema/any-query
```

Comment:
I renamed `::qp.schema/query` to `::qp.schema/any-query` to make it clearer it means either legacy MBQL or MBQL 5